src/common/pico_stdlib/include/pico/stdlib.h

```diff
@@ -56,6 +56,10 @@ extern "C" {
 #include "pico/stdio_semihosting.h"
 #endif
 
+// PICO_CONFIG: PICO_DEFAULT_LED_PIN, Optionally defined a pin that drives a regular LED on the board, default=undefined, group=pico_stdlib
```

Comment:
See e.g. https://github.com/raspberrypi/pico-sdk/pull/183/commits/9b05c241a86e46ad1e8aa8831e159440e23bc232
I wonder if it's better to simply omit `default`, rather than having `default=undefined` ?

Reply:
the intention here is to indicate that the value is omitted rather than having a default value